File: t2/SEP3T2API/SEP3T2API/SEP3T2GraphQL/Models/GuestRegistrationRequest.cs
using System.ComponentModel.DataAnnotations;

namespace SEP3T2GraphQL.Models
{
    public class GuestRegistrationRequest
    {
        [Required]
        public int Id { get; set; }
        [Required]
        public int StudentNumber { get; set; }
        [Required]
        public string StudentIdImage { get; set; }
        [Required]
        public RequestStatus Status { get; set; } = RequestStatus.NotAnswered ;
    }
}

File: t3/viabnb.tier3.dataserver.soapws/src/main/java/dk/viabnb/sep3/group6/dataserver/rest/t3/models/GuestRegistrationRequest.java
package dk.viabnb.sep3.group6.dataserver.rest.t3.models;

public class GuestRegistrationRequest
{
  private int id;
  private int studentNumber;
  private String studentImage;
  private RequestStatus status = RequestStatus.NotAnswered;

  public GuestRegistrationRequest(int id, int studentNumber,
      String studentImage, RequestStatus status)
  {
    this.id = id;
    this.studentNumber = studentNumber;
    this.studentImage = studentImage;
    this.status = status;
  }

  public int getId()
  {
    return id;
  }

  public void setId(int id)
  {
    this.id = id;
  }

  public int getStudentNumber()
  {
    return studentNumber;
  }

  public void setStudentNumber(int studentNumber)
  {
    this.studentNumber = studentNumber;
  }

  public String getStudentImage()
  {
    return studentImage;
  }

  public void setStudentImage(String studentImage)
  {
    this.studentImage = studentImage;
  }

  public RequestStatus getStatus()
  {
    return status;
  }

  public void setStatus(RequestStatus status)
  {
    this.status = status;
  }

}
